lib/telephone_number/formatter.rb

```diff
@@ -1,13 +1,56 @@
 module TelephoneNumber
-  module Formatter
+  class Formatter
+    extend Forwardable
```

Comment:
I don't believe that will work. Since we're not inheriting, those methods that are extended will never actually be included in the lookup chain.